turkiye/turkiye_earthquake_legitimate_iocs.csv.csv

```diff
@@ -0,0 +1,25 @@
+﻿type,indicator,classification,references
+domain,yikilanbinalar.com.tr,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,enkazbildirim.org,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,howtohelpturkey-syriaearthquakevictims.org,legitimate,
+domain,hepsiemlak.com,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,kizilay.org.tr,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,misafirimol.trair.com.tr,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,depremihtiyac.com,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,akut.org.tr,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,adresdeprem.com,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,earthquaketurkey.com,legitimate,

```

Comment:
why do we know this is legit? just older registration?

Reply:
"Recently registered domain on 2023-02-08. Domain contains news related to the earthquake which happened in Turkey on February 2023. No links or references to funding or anything which could lead to identifying suspicious behaviour at the time of the investigation (2023-02-10)."

Reply:
ok